scalecube-configuration/src/main/java/io/scalecube/configuration/repository/ConfigurationRepository.java

```diff
@@ -41,7 +41,17 @@
    * @param doc the document to save.
    * @return Document instance by a given key.
    */
-  Mono<Document> save(String tenant, String repository, Document doc);
+  Mono<Document> createEntry(String tenant, String repository, Document doc);
+
+  /**
+   * Update a key in a tenant repository by key.
+   *
+   * @param tenant namespace of the repository.
+   * @param repository name to update key.
+   * @param doc the document to update.
+   * @return Document instance by a given key and version.
+   */
+  Mono<Document> updateEntry(String tenant, String repository, Document doc);
```

Comment:
Let's do not change the internal interface for the sake of public API (service API) changes.

Reply:
I changed it for more visibility not for just public API, but you idea it good too.

Reply:
I think @dmytro-lazebnyi wanted to remove unsupported method at all. Correct me if it's not so.

Reply:
@segabriel
`ConfigurationRepository` interface has to leave unchanged since it is an internal component and not related to public service API.
@stingion please revert all changes of this interface.

Reply:
It' requires reverting all and remake again